core/tests/protractor/learnerDashboard.js

```diff
@@ -91,9 +88,9 @@ describe('Learner dashboard functionality', function() {
     player.submitAnswer(
       'MultipleChoiceInput', 'Those were all the questions I had!');
     player.submitAnswer('Continue', null);
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     browser.waitForAngular();
-    element(by.css('.protractor-test-completed-section')).click();
+    learnerDashboardPage.completedProfileTest();
```

Comment:
If this represents a click, the name of the method should indicate that. E.g. learnerDashboardPage.moveToCompletedSection() or learnerDashboardPage.openCompletedSection() (or whatever the click is actually doing semantically; I can't tell from the name that's being given to it here).

Reply:
done, I have renamed all occurrences of such names.